ui/perfherder/graphs/GraphTooltip.jsx

```diff
@@ -4,7 +4,10 @@ import countBy from 'lodash/countBy';
 import moment from 'moment';
 import { Button } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import {
+  faExclamationCircle,
+  faTimes,
+} from '@fortawesome/free-solid-svg-icons';
```

Comment:
This is probably not applicable here, but when I was looking for a solution to something else, I came across ``react-tooltip`` that allows you to put any ``HTML`` into it.  I had wondered if that would work for this.  Just something to keep in your back pocket in case you need a fancier tooltip at some point.  It tracks the pointer pretty well.  But not clear if it will lock when you click like this one does..  :)

Reply:
Interesting, I'll check it out. The locking happens by keeping a data point "active", via the graphing library's external mutation. Since I'm hooking into their tooltip component, that work to display it happens under the hood. I'm just swapping in my customized component instead of using their very basic default tooltip.